internal/server/iigointernal/sanctionevaluator.go

```diff
@@ -0,0 +1,93 @@
+package iigointernal
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/pkg/errors"
+	"gonum.org/v1/gonum/mat"
+)
+
+func evaluateSanction(sanction rules.RuleMatrix, localVariableCache map[rules.VariableFieldName]rules.VariableValuePair) shared.Resources {
```

Comment:
while the rest of the functions in this file look simple enough and likely don't need it, this function seems complicated enough to justify at least one testcase. Apologies if you have already made a test case for it, i can't seem to find it

 Plus a comment on how it works would be appreciated

Reply:
Sure, added test, I'll add some testing for the rest of judiciary while I'm at it, might as well get that done now